charts/seed-monitoring/charts/prometheus/rules/apiserver-connectivity-check.rules.yaml

```diff
@@ -0,0 +1,24 @@
+groups:
+- name: apiserver-connectivity-check.rules
+  rules:
+  - alert: APIServerUnreachableFromShoot
+    expr: probe_success{job=~"blackbox-exporter-k8s-service-check-[0-9]"} != 1
+    for: 5m
+    labels:
+      service: apiserver-connectivity-check
+      severity: critical
+      type: shoot
+    annotations:
+      summary: API Server is not reachable from shoot.
+      description: The API Server is unreachable from the shoot.
+
+  - alert: ConnectionToAPIServerSlow
```

Comment:
We need an alert with `severity: warning` when at least 10% of the connection attempts need more then 10ms to complete. This alert can fire immediately because we should execute this test only every 5m.

Then we need another alert with `severity: warning`  when we have at least ONE of the 20 connection attempts aborts after `128s` (tcp connection reset of the loadbalancers). This alert can fire immediately because we should execute this test only every 5m.

Then we need for the another alert with `severity: critical` for the test which checks if the apiserver is reachable via the `kubernetes` service (for this we need additional scrape config - see comment below). This alert can fire after `2-3m`, because the test should be executed every `30s`.

Reply:
done